packages/web/scripts/workers-site/index.js

```diff
@@ -134,9 +134,10 @@ class SEOHandlerBody {
 }
 
 class SEOHandlerHead {
-  constructor(pathname, discoveryNode) {
+  constructor(pathname, discoveryNode, hostname = 'https://audius.co') {
```

Comment:
can this be pulled from .env?

Reply:
Actually prefer using `url.host` so that ngrok can work for testing

I removed this default though, it's unnecessary, and updated to use `url.host` instead of `origin` since the latter seemed to be `http` for some reason

Reply:
yeah agree. There's already some hardcoded https://audius.co in there though :(